### PR TITLE
First round of little fixes suggested by the audit report

### PR DESCRIPTION
**Pull request by gregory-latinier**

### CODE REVIEW

File: packages/api/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (ms10398) [inline packages/api/LICENSE]: Shouldn't this be in txt format as suggested on other LICENSE file by @nothingismagick

File: packages/browser-extension/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
```
Comment (ms10398) [inline packages/browser-extension/LICENSE]: Same

File: packages/auth-web/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (ms10398) [inline packages/auth-web/LICENSE]: Same

Comment (nothingismagick) [reply packages/auth-web/LICENSE]: No, there should not be a .txt suffix, this is correct.

File: packages/api/LICENSE (hunk 1 above)
Comment (nothingismagick) [reply packages/api/LICENSE]: No, there should NOT be a .txt suffix

File: packages/auth-web/LICENSE (hunk 3 above)
Comment (ms10398) [reply packages/auth-web/LICENSE]: Oops I realised you said to do vice versa

**Pull request merged**